architecture/core_concepts/routes.adoc

```diff
@@ -322,9 +322,19 @@ In order for services to be exposed externally, an {product-title} route allows
 you to associate a service with an externally-reachable host name. This edge
 host name is then used to route traffic to the service.
 
-When two routes claim the same host, the oldest route wins. If additional routes
-with different path fields are defined in the same namespace, those paths will be
-added. If multiple routes with the same path are used, the oldest takes priority.
+When multiple routes from different namespaces claim the same host,
```

Comment:
Perhaps I am misunderstanding how it works... but the way I read the code, if I have:

Namespace 1:

- Route 1 Host "foo.com" Path "/foo" [Created 1/1/2001]
- Route 2 Host "foo.com" Path "/" [Created 9/1/2001]


Namespace 2:

- Route 3 Host "foo.com" Path "/bar" [Created 2/1/2001]
- Route 4 Host "foo.com" Path "/foo" [Created 3/1/2001]

If all of those are in effect, then only the routes from Namespace 1 will be able to apply any routes to "foo.com" because Route 1 was created first.  So requests to /foo and below go to route 1, everything else goes to route 2.

Now... if I delete Route 1, then since Route 3 is next oldest, the routes in namespace 2 will take effect for foo.com.  So /bar and below goes to route 3, /foo and below goes to route 4, and everything else gets an HTTP not found.

Is my understanding correct?